### DISCUSSION TITLE
Login as member for non Super Admins

### DISCUSSION THREAD

**Discussion by chrisdaviesweb** [ExpressionEngine/ExpressionEngine#3907] [Feature Requests]
I often find use cases, where clients need to log in as members, but aren't 'Super Admin'. I always create an 'Admin' role for clients, removing access to developer areas etc.

Can a setting be added to 'CP Access' when creating/editing a role, allowing to toggle allow/disallow for 'Login as member' functionality. Perhaps as a checkbox with the following?

![members](https://github.com/ExpressionEngine/ExpressionEngine/assets/51822011/de2b7926-ddab-4424-a3bf-33daf7648b15)

**Comment by twist-id** [comment]
I have a similar issues and it would be handy to have a more default editor role as standard in the CP that allows the client to edit and add content etc along with creating members but that's it.

**Comment by intoeetive** [comment]
While we probably should not allow non-superadmins "login as" functionality on the members that have CP access - as that potentially could give them higher privileges, this would probably be safe thing to do on users that have frontend access only.

@TomJaeger so say his word on it however

(also moving to discussions as this is feature request)

**Reply by chrisdaviesweb** [reply]
Agree - Just 'Site Index'.

**Comment by intoeetive** [comment]
Looking into this again, if we were to ever implement this, we´d need some "permission level" check, so that person does not log in as someone with higher permissions. Even if we redirect them to front-end - there might be restricted functional there.
